frontend/src/header/Header.jsx

```diff
@@ -23,7 +25,7 @@ export default function Header() {
   }
```

Comment:
This line : ` <Button key="address" type="dashed" disabled>0x...</Button>` can update `0x...` to be the address of the current wallet or a truncated version if it

Reply:
Is there a way to automatically populate the address if I'm already connected to the site, currently shows connect button even though I'm connected. Once I press the connect button then it shows I'm connected.

Reply:
Looks good tho!